imports/ui/components/modals/foundKeystore.js

```diff
@@ -5,10 +5,13 @@ import '/imports/ui/components/alert/modalAlert.js'
 
 Template.foundKeystore.events({
   'click #btn-foundKeystore-login' (event, instance) {
+    console.log('cliccato')
     showModal('login')
   },
   'click #btn-foundKeystore-cancel' (event, instance) {
     hideModal()
     showGlobalAlert(_(`You are now anonymous, of course you can <strong><a data-closealert data-showmodal="login">log in</a></strong> at any moment`))
+    // This one is used to hide/show the foundKeystore modal
+    window.sessionStorage.setItem('navigation', 'anonymous')
```

Comment:
I think a function that wraps setting of this storage property might be a good idea. It will reduce the need for a comment every time it is done (assuming the function is named in a helpful manner) and it will be easier to update if the logic used to hide/show this modal ever changes. Just a thought.